src/services/match.ts

```diff
@@ -12,10 +12,10 @@ export const createMatch = (matchPayload: Omit<Match, 'id' | 'participants'>) =>
     })
     .catch((err) => {
       if (err.status === 400) {
-        throw new Error('Incorrect match format')
+        throw new Error(err.getErrorMessage)
```

Comment:
Comment about errors in general: we should handle them through react query and use a generic pop up to give feedback to the user

Reply:
I've created a new generic function to handle the Axios error and console them. I also developed a new Alert component that we can use to give feedback to the user.